### ISSUE TITLE
do we want df included in output where available?

### ISSUE THREAD

**Issue by bbolker** [bbolker/broom.mixed#3]
**why not:**

- clutters up output
- not available in some cases
- wasn't included in original `broom` tidiers (for whatever reason)

**why:**

- might want to see it sometimes
- it's easy enough to drop with a `select(-df)`
- `dplyr::bind_rows()` makes incommensurate columns in different sets of output fairly harmless

**Comment by bbolker** [comment]
also see https://github.com/tidyverse/broom/issues/212

**Comment by paul-buerkner** [comment]
I think it is generally ok, not to have the same columns for all tidies. I can assume, we will run across similar issues with other special columns, which might not be available / desired in all models. Take p-values as an example (not sure if they are anywhere in the tidiers): We could easily approximate p-values in Bayesian models, but most of Bayesians statisticians would be opposed to computing them at all.

**Comment by bbolker** [comment]
I agree (although note that `MCMCglmm` *does* provide p-values; I agree, I don't like them ...).

I mostly raised this because it reverses a presumably deliberate design choice made in the original `broom` tidiers ...

**Comment by bwiernik** [comment]
I don't think there was a design decision here in `broom`—I think it's just that for many models no df is reported because they are all 1. But, for example, `tidy.aov()` does have a `df` column (and a `sumsq` and `meansq`) column.
